encoding.bs

```diff
@@ -216,15 +216,22 @@ happening.
  <var>encoderDecoderInstance</var> is a <a for=/>decoder</a> instance, and "<code>fatal</code>"
  otherwise.
 
+ <li><p>Assert: If <var>encoderDecoderInstance</var> is an <a for=/>encoder</a> instance,
+ <var>token</var> is not a <a>surrogate</a> code point.
+
  <li><p>Let <var>result</var> be the result of running <var>encoderDecoderInstance</var>'s
  <a>handler</a> on <var>input</var> and <var>token</var>.
 
  <li><p>If <var>result</var> is <a>continue</a> or <a>finished</a>, return
  <var>result</var>.
 
  <li><p>Otherwise, if <var>result</var> is one or more
- <a>tokens</a>, <a>push</a>
- <var>result</var> to <var>output</var>.
+ <a>tokens</a>:
+  <ol>
+   <li><p>Assert: If <var>encoderDecoderInstance</var> is a <a for=/>decoder</a>
+   instance, <var>result</var> does not contain any <a>surrogate</a> code points.
+   <li><p><a>Push</a> <var>result</var> to <var>output</var>.
+  </ol>
```

Comment:
Can you format this as the following list item? In particular, now the list item contains multiple children, the `<p>` needs to be on its own line. Also it should be "surrogates" (no code points, as that's part of its definition).